src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HPack/Huffman.cs

```diff
@@ -268,30 +267,12 @@ internal class Huffman
             (0b11111111_11111111_11111111_11111100, 30)
         };
 
-        private static readonly (int codeLength, int[] codes)[] s_decodingTable = new[]
+        private static readonly short[,] s_decodingArray;
```

Comment:
Shouldn’t this just be:

```c#
private static readonly short[,] s_decodingArray = BuildDecodingArray();
```

without the explicit static constructor?

Reply:
Changed